dashboard/app/dsl/script_dsl.rb

```diff
@@ -42,10 +43,16 @@ def initialize
 
   string :wrapup_video
 
-  def stage(name, flex = nil)
+  # Can either provide a flex_category as an optional param, or an options hash
```

Comment:
I wonder if we should lock to always using an options hash, and update existing scripts.